skeleton/templates/cdk/src/api-stack.ts.mustache

```diff
@@ -162,8 +165,11 @@ export class ApiStack extends Stack {
             assignPublicIp: true,
             publicLoadBalancer: true,
             healthCheckGracePeriod: Duration.seconds(10),
+            sslPolicy: SslPolicy.RECOMMENDED_TLS,
```

Comment:
What exactly is this changing?

Reply:
by default cdk sets this to the crummy setting of ELBSecurityPolicy-2016-08
https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html

RECOMMENDED_TLS should use the recommended version for the currently used cdk version.